Ssmpu/Memory_Protection_Unit.adoc

```diff
@@ -102,12 +121,12 @@ MPU does not permit supervisor mode to execute instructions from U-mode regions,
 === Priority and Matching Logic
 The PMP checks only take effect after the memory access passes the MPU permission checks. An M-mode access will not be checked by MPU property.
 
-Like PMP entries, MPU entries are also statically prioritized. The lowest-numbered MPU entry that matches any byte of an access determines whether that access succeeds or fails. The matching MPU entry must match all bytes of an access, or the access fails, irrespective of the S, R, W, and X bits.
+Like PMP entries, MPU entries are also statically prioritized. The lowest-numbered MPU entry that matches any byte of an access determines whether that access is allowed or fails. The matching MPU entry must match all bytes of an access, or the access fails, irrespective of the S, R, W, and X bits.
```

Comment:
what do you mean by "any byte of an access" ? entries are matched using addresses, did you mean any byte of an address ?

Reply:
It is any bytes represented by an address and the length (indicated by the load/store instructions).